### ISSUE TITLE
Polyphone 1.9 Compiled on Ubuntu 16.04 Crashes with Stack Trace

### ISSUE THREAD

**Issue by zicklag** [davy7125/polyphone#35]
Hi, I am trying to get Polyphone running on my Ubuntu 16.04 machine and while I am able to successfully compile it and create the Polyphone binary, running the program yields a stack trace:

```
*** Error in `polyphone': realloc(): invalid pointer: 0x000000000091e400 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x777e5)[0x7efbfbd927e5]
/lib/x86_64-linux-gnu/libc.so.6(realloc+0x348)[0x7efbfbd9fa08]
/usr/lib/x86_64-linux-gnu/libQt5Core.so.5(_ZN9QListData7reallocEi+0x1f)[0x7efbfa9439cf]
/usr/lib/x86_64-linux-gnu/libQt5Core.so.5(_ZN9QListData6appendEi+0x81)[0x7efbfa943aa1]
/usr/lib/x86_64-linux-gnu/libQt5Core.so.5(+0x1d6d78)[0x7efbfaa0fd78]
/usr/lib/x86_64-linux-gnu/libQt5Core.so.5(_Z21qRegisterResourceDataiPKhS0_S0_+0x2e6)[0x7efbfaa0bb16]
/usr/lib/x86_64-linux-gnu/libQt5Core.so.5(+0x7bcc3)[0x7efbfa8b4cc3]
/lib64/ld-linux-x86-64.so.2(+0x106ba)[0x7efbff4e96ba]
/lib64/ld-linux-x86-64.so.2(+0x107cb)[0x7efbff4e97cb]
/lib64/ld-linux-x86-64.so.2(+0xc6a)[0x7efbff4d9c6a]
...I Truncated the Memory Map...
```
Is there anything that I might be able to try to fix this or any other information that would be useful to help identify the problem?

**UPDATE:** I have the same issue when compiling Polyphone 1.8 and the master version.

**Comment by zicklag** [comment]
I have both Qt4 and Qt5 dev packages installed on my system. I can't tell for sure, but it looks like it might be building Polyphone with Qt4 and attempting to run Polyphone with Qt5. Is there a way to specify the version of Qt to use when building/running?

**Comment by zicklag** [comment]
Aha! I figured it out. I had to clear the build cache by removing everything in `RELEASE` and use the Qt5's `qmake` binary which was in `/usr/lib/x86_64-linux-gnu/qt5/bin/qmake`. After running that then `make` the binary successfully ran. I still don't know why it was trying to use the `libQt5Core.so.5` lib when I use Qt4 version of qmake, but at least I found a fix. Can't wait to try it out! :smiley:
